src/components/FloatingDropdown/FloatingDropdownBase.st.css

```diff
@@ -0,0 +1,83 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color;
+}
+
+:import {
+    -st-from: "../Button/Button.st.css";
+    -st-default: TPAButton;
+}
+
+:import {
+    -st-from: "../Input/Input.st.css";
+    -st-default: TPAInput;
+    -st-named: backgroundColor, colorDisabled, colorPlaceholder, color as inputColor;
+}
+
+.root {
+    -st-extends: TPAButton;
+    -st-mixin: TPAButton(
+        MainBackgroundColor 'value(backgroundColor)',
+        MainTextColor 'value(inputColor)'
+    );
+    display: flex;
+    cursor: pointer;
+    box-sizing: border-box;
+    outline: none;
+}
+
+.arrowIcon {
+    color: value(inputColor);
+}
+
+.text {
+    -st-mixin: TPAText(
+        MainTextFont "{theme: 'Body-M', size: '16px'}"
```

Comment:
Hi, @jonathanadler 

Docs https://zeroheight.com/7sjjzhgo2/p/05a744-05-selections/b/46537f/t/36cac4

Says that I should use `Paragraph 2`:
![image](https://user-images.githubusercontent.com/1897702/78213458-662aaa80-74bb-11ea-83eb-2ccb7f235ab8.png)

I noticed that <Dropdown> has the same wiring:
![image](https://user-images.githubusercontent.com/1897702/78213514-8f4b3b00-74bb-11ea-9002-1de651d25c66.png)


And uses `-st-mixin: TPAText(MainTextFont "{theme: 'Body-M'`

Anyway, where is mapping between `Paragraph 2` and `Body-M` happening?

Reply:
That's a great question, and I hope we'll be able to make this more clear in the near future.
Our designers use `Paragraph 2` because that's the name that appears in the editor.
But the font variable is called `Body-M`, which quite confusing 😖 
The conversion is done by this tool we use:
https://github.com/wix/wix-style-processor
(but TPA's also use this tool instead: https://github.com/wix-incubator/tpa-style-webpack-plugin, which is pretty much the same, but works as a webpack plugin, and comes with yoshi apps).

As I said, I hope to have some Stylable constants in the future, so that contributors will only need to write:
```css
:import {
    -st-from: "../../common/typography.st";
    -st-named: paragraph_2;
}

.selector {
  font: value(paragraph_2);
}
```

Or a mixin, or something like that.

I hope this answered your question 🤔 .

Reply:
Yes, thanks :)